fxa-auth-db-server/docs/API.md

```diff
@@ -62,6 +62,7 @@ The following datatypes are used throughout this document:
     * createDevice              : `PUT /account/:id/device/:deviceId`
     * updateDevice              : `POST /account/:id/device/:deviceId/update`
     * deleteDevice              : `DEL /account/:id/device/:deviceId`
+    * deviceFromTokenVerificationId : `GET /account/:id/device/fromTokenVerificationId/:tokenVerificationId`
```

Comment:
The API exposed by this server is a little haphazard, but for consistency, I think we should follow the scheme where each path component names the thing that you're returning or querying by, which would make this something like:

* `GET /account/:id/tokens/:tokenVerificationId/device`

It's then easy to imagine sibling routes like `tokens/:tvid/sessionToken`. What do you think?